src/OpenLoco/Windows/TitleMenu.cpp

```diff
@@ -275,6 +275,7 @@ namespace OpenLoco::Ui::Windows::TitleMenu
             OpenLoco::Gfx::drawImage(context, x, y, image_id);
         }
 
+        /*
```

Comment:
Not a fan of having commented-out code in the repo, as it doesn't get checked if it compiles. Can we add a run-time option for showing/not showing it?